Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -537,6 +537,9 @@ RefPtr<ServiceWorkerFetchTask> NetworkConnectionToWebProcess::createFetchTask(Ne
 
 void NetworkConnectionToWebProcess::scheduleResourceLoad(NetworkResourceLoadParameters&& loadParameters, std::optional<NetworkResourceLoadIdentifier> existingLoaderToResume)
 {
+    if (UNLIKELY(!m_networkProcess->allowsFirstPartyForCookies(m_webProcessIdentifier, loadParameters.request.firstPartyForCookies())))
```

Comment:
We should avoid calling allowsFirstPartyForCookies() twice. Just store the result in a local variable.